donkeycar/parts/tub_v2.py

```diff
@@ -52,6 +52,8 @@ def write_record(self, record=None):
                     contents[key] = int(value)
                 elif input_type == 'boolean':
                     contents[key] = bool(value)
+                elif input_type == 'nparray':
+                    contents[key] = (np.array(value)).tolist()
```

Comment:
If `type(value) == np.array`, then you don't need the `np.array(...)` around it.